specification/search/data-plane/Azure.Search/servicereadme.md

```diff
@@ -0,0 +1,247 @@
+
+``` yaml $(tag) == 'package-2019-05-searchservice'
+input-file:
+- stable/2019-05-06/searchservice.json
+```
+
+### Tag: package-2019-05-searchindex
+
+These settings apply only when `--tag=package-2019-05-searchindex` is specified on the command line.
+
+``` yaml $(tag) == 'package-2019-05-searchindex'
+input-file:
+- track1/stable/2019-05-06/searchservice.json
+```
+
+---
+# Code Generation
+
+!!! READ THIS !!!
+This swagger is ready for C# and Java.
+!!! READ THIS !!!
+
+## Java
+
+These settings apply only when `--java` is specified on the command line.
+Please also specify `--java-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-java clone>`.
+
+``` yaml $(java)
+title: SearchServiceRestClient
+output-folder: $(java-sdks-folder)/search
+namespace: com.azure.search
+java: true
+sync-methods: none
+add-context-parameter: true
+generate-client-interfaces: false
+custom-types-subpackage: implementation.models
+custom-types: AnalyzeResult,SuggestRequest,AutocompleteRequest
+license-header: |-
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT License.
+  Code generated by Microsoft (R) AutoRest Code Generator.
+  Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+require: ./autorest-custom-directives.md
+
+vararg-properties: >-
+  CorsOptions.allowedOrigins
+
+directive:
+    # Add static Collection<DataType> method to DataType
+    - from: DataType.java
+      where: $
+      transform: >-
+          return $
+          .replace(/(public static final DataType EDM_COMPLEX_TYPE = fromString\("Edm.ComplexType"\);)/g, "$1\n\n    /**\n     * Returns a collection of a specific DataType\n     * @param dataType the corresponding DataType\n     * @return a Collection of the corresponding DataType\n     */\n    @JsonCreator\n    public static DataType Collection(DataType dataType) {\n        return fromString(String.format(\"Collection(%s)\", dataType.toString()));\n    }")
+
+    # Workaround to fix bad host path parameters
+    - from:
+          - SkillsetsImpl.java
+          - DatasetsImpl.java
+          - DataSourcesImpl.java
+          - IndexersImpl.java
+          - IndexesImpl.java
+          - SynonymMapsImpl.java
+      where: $
+      transform: >-
+          return $
+          .replace(/(this.getSearchServiceName)/g, "this.client.getSearchServiceName")
+          .replace(/(this.getEndpoint)/g, "this.client.getEndpoint")
+          .replace(/(this.getSearchDnsSuffix)/g, "this.client.getSearchDnsSuffix")
+
+    # Replace VoidResponse with SimpleResponse<Void>
+    - from:
+          - SkillsetsImpl.java
+          - DatasetsImpl.java
+          - DataSourcesImpl.java
+          - IndexersImpl.java
+          - IndexesImpl.java
+          - SynonymMapsImpl.java
+      where: $
+      transform: >-
+          return $
+          .replace(/(import com.azure.core.http.rest.VoidResponse;\n)/g, "")
+          .replace(/(VoidResponse)/g, "SimpleResponse<Void>")
```

Comment:
`VoidResponse` isn't a type in the Java repo anymore, this was removed before GA of Azure Core, is this still needed?

Reply:
I will remove and test